src/main/resources/templates/templateOpportunityNotification.xml

```diff
@@ -52,10 +54,14 @@
                 <div>
                     <span class="tempo-text-color--secondary">Close Date:</span>
                     <span>${entity['OpportunityNotification'].current.opportunity.CloseDate ! "-"}</span>
-                    <span class="tempo-text-color--secondary">Probability:</span>
+                    <span class="tempo-text-color--secondary">&#160;&#160;&#160;Probability:</span>
```

Comment:
I suggest that you write the empty spaces instead of using this ASCII code. Firstly because it is more readable. Secondly because all other integrations are doing that in this way.